src/main/java/io/github/cottonmc/energy/CottonEnergy.java

```diff
@@ -1,22 +1,24 @@
 package io.github.cottonmc.energy;
 
 
+import io.github.cottonmc.cotton.Cotton;
+import io.github.cottonmc.energy.api.EnergyType;
+import io.github.cottonmc.energy.api.DefaultEnergyTypes;
 import net.fabricmc.api.ModInitializer;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.DefaultMappedRegistry;
+import net.minecraft.util.registry.ModifiableRegistry;
+import net.minecraft.util.registry.Registry;
 
 public class CottonEnergy implements ModInitializer {
 
+	public static final ModifiableRegistry<EnergyType> ENERGY_REGISTRY = new DefaultMappedRegistry("energy");
```

Comment:
also, let's keep the static field a Registry<EnergyType> instead of a ModifiableRegistry, and users can go through the preferred Mojang registration flow.

Reply:
It needs to be a ModifiableRegistry to be able to be registered through Registry.REGISTRIES.

Reply:
and is the "cotton:empty" for the empty value or the identifier of the registry? I need to double-check.

Reply:
Ah, it is for the empty value.